float32[] focus_distance_vector
float32[] focal_length_vector
float32[] aperture_vector
float32[] yaw_vector
float32[] times_vector
float32[] pitch_vector
airsim_ros_pkgs/MoveOnPath move_on_path_msg